### PR TITLE
Fix nxp 25604 cancels3 upload

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

**Comment by guirenard** [comment]
Depends on https://github.com/nuxeo/nuxeo-web-ui/pull/438

File: nuxeo-core-binarymanager-s3-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-s3-direct-upload/nuxeo-s3-direct-upload.html [hunk 1]
```diff
@@ -127,11 +128,26 @@
       }.bind(this));
     };
 
+    S3Provider.prototype.hasAbort = function() {
```
Comment (Gabez0r) [inline nuxeo-core-binarymanager-s3-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-s3-direct-upload/nuxeo-s3-direct-upload.html]: For us to take advantage of this on elements using `Nuxeo.UploaderBehavior`, we'd have to add to this behavior a respective method:
```
 hasAbort() {
        return this._instance.hasAbort();
}
``` 
However, I don't think we need this yet if we take https://github.com/nuxeo/nuxeo-web-ui/pull/438#discussion_r246051800 into account.

File: nuxeo-core-binarymanager-s3-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-s3-direct-upload/nuxeo-s3-direct-upload.html [hunk 2]
```diff
@@ -127,11 +128,26 @@
...
+      return true;
+    };
+
     S3Provider.prototype.hasProgress = function() {
       return true;
     };
 
+    S3Provider.prototype.abort = function(file) {
```
Comment (Gabez0r) [inline nuxeo-core-binarymanager-s3-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-s3-direct-upload/nuxeo-s3-direct-upload.html]: Similar to the above comment, we aren't making use of this method anywhere else, so I wouldn't make it public.

**Comment by Gabez0r** [comment]
👍

**Pull request merged**